tests/sentry/api/test_event_search.py

```diff
@@ -721,6 +818,48 @@ def test_numeric_filter(self):
             )
         ]
 
+    def test_numeric_in_filter(self):
+        assert parse_search_query("project_id:[500,501,502]") == [
```

Comment:
Are project slugs resolved when they are used in an IN condition?

Reply:
Does the parser usually resolve these to slugs, or is it at a later step?

Reply:
project slugs get resolved via a transform statement added to query columns via `resolve_field_list`
something like:
```sql
transform(project_id, [1,2,3], ["sentry", "default", "blah"])
```